recipes/hfsm2/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from conan import ConanFile
+from conan.tools.files import copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.layout import basic_layout
+from conan.tools.files import get
+
+required_conan_version = ">=2.0.16"
```

Comment:
```suggestion
required_conan_version = ">=1.54.0"
```

Conan 1.x support is still mandatory in CCI.